CveXplore/core/database_maintenance/sources_process.py

```diff
@@ -98,14 +99,14 @@ def process_the_item(self, item: dict = None):
 
         sha1_hash = hashlib.sha1(
             cpe["cpeName"].encode("utf-8")
-            + item["cpeName"].split(":")[5].encode("utf-8")
+            + split_cpe_name[5].encode("utf-8")
```

Comment:
Black formatting. See the failing workflow. Combine with the previous line:
```python
            cpe["cpeName"].encode("utf-8") + split_cpe_name[5].encode("utf-8")
```

Reply:
fixed by https://github.com/cve-search/CveXplore/pull/269/commits/8ad6c647c7e31822d6c2a9511bc9bb4f9663ea05